src/pages/set-password.tsx

```diff
@@ -70,6 +69,7 @@ export const SetPasswordPage: React.FC<SetPasswordProps> = ({
       setLoading(true);
       if (strengthResult.meetsAllStrengthRequirements) {
         await submit(password);
+        chrome.runtime.sendMessage('onboarding-complete');
```

Comment:
In the above example, you'll run into type errors as `onboarding-complete` isn't typed anywhere. You'll need to add this to the `VaultActions` types.

It'd also be better to follow the same `Message` format, which is to use an object with various properties.

At some point I think we'll want to avoid using the `chrome` global directly like this, rather in wrapper functions, but wouldn't worry about this for now.

Reply:
> In the above example, you'll run into type errors as `onboarding-complete` isn't typed anywhere. You'll need to add this to the `VaultActions` types.
> 
> It'd also be better to follow the same `Message` format, which is to use an object with various properties.
> 

Good point, will change. 👍 

> At some point I think we'll want to avoid using the `chrome` global directly like this, rather in wrapper functions, but wouldn't worry about this for now.

I agree, struggled a bit with how at this point ...that is when I found the proxy redux store pattern where there is just shared state b/w the background script and react components.